noetic/distribution.yaml

```diff
@@ -4744,11 +4745,13 @@ repositories:
       - jsk_3rdparty
       - julius
       - julius_ros
+      - laser_filters_jsk_patch
       - libcmt
       - libsiftfast
       - lpg_planner
       - mini_maxwell
       - nfc_ros
+      - nlopt
```

Comment:
nlopt appears to be available in Ubuntu Focal. Please use a rosdep key instead:

https://launchpad.net/ubuntu/focal/arm64/libnlopt-dev

Unfortunately packages.ubuntu.com already removed Focal listings, so Jammy is the earliest ig can show. https://packages.ubuntu.com/jammy/libnlopt-dev

Reply:
nloopt is already exists at https://github.com/ros/rosdistro/pull/45761/files, so removed from release